src/libs/actions/Report.ts

```diff
@@ -2043,6 +2058,13 @@ function navigateToSystemChat() {
 /** Add a policy report (workspace room) optimistically and navigate to it. */
 function addPolicyReport(policyReport: ReportUtils.OptimisticChatReport) {
     const createdReportAction = ReportUtils.buildOptimisticCreatedReportAction(CONST.POLICY.OWNER_EMAIL_FAKE);
+    const policy = getPolicy(policyReport.policyID);
+
+    const missingFields: ReportFieldsViolations = {};
+
+    Object.values(policy?.fieldList ?? {}).forEach((field) => {
+        missingFields[field.fieldID] = {};
```

Comment:
Can we add a comment here to indicate that there's no additional data required for this specific violation and so the data is just an empty json